### PR TITLE
Add Prometheus Helm chart and basic metrics.

### PR DESCRIPTION
**Pull request by jsha**
Note: This does not yet use a regional GCE Persistent Disk even though that was recommended in #194. I wanted to get something published that was simple and demonstrated actual metrics flowing through. Plumbing through a setting to use the regional disk may take longer and can be done in parallel.

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -293,3 +293,19 @@ output "specific_manifests" {
...
+
+  set {
+    name  = "server.ingress.enabled"
+    value = "true"
+  }
+
+  # Values available for overriding are listed in https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml
+  set {
+    name  = "server.ingress.hosts[0]"
+    value = "prometheus.prio.crud.net"
```
Comment (tgeoghegan) [inline terraform/main.tf:309]: Should this value be interpolated from something per-environment?

**Review by tgeoghegan — CHANGES_REQUESTED** [review]
If not a persistent disk, where are metrics stored right now? The filesystem of the container that Prometheus runs in?

Would you mind filing issues for follow on work that you're aware of like configuring durable storage, setting up Grafana, and finding a Rust Prometheus client to make it easier to hand this work off when we all come back from the Native American Heritage Day long weekend?

**Comment by jsha** [comment]
> If not a persistent disk, where are metrics stored right now?

They're stored on a persistent disk (which is created by the Helm chart). It's just not regional (afaict).

> Would you mind filing issues for follow on work that you're aware of like configuring durable storage, setting up Grafana, and finding a Rust Prometheus client to make it easier to hand this work off when we all come back from the Native American Heritage Day long weekend?

Will do!

File: terraform/main.tf (hunk 1 above)
Comment (jsha) [reply terraform/main.tf]: Good catch. I removed this for now; we don't actually want to export Prometheus by default.

**Review by tgeoghegan — APPROVED** [review]

**Pull request merged**